cdhweb/static_src/styles.scss

```diff
@@ -60,6 +61,7 @@
 @import './global/components/block-quote.scss';
 @import './global/components/download.scss';
 @import './global/components/search-form.scss';
+@import './global/components/pagination.scss';
```

Comment:
Kept separate as it looks like other things will be paginated later on, e.g. news section